src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/StatusAlert.jsx

```diff
@@ -54,12 +54,19 @@ export default function StatusAlert({ appointment, facility }) {
         <strong>We’ve scheduled and confirmed your appointment.</strong>
         <br />
         <div className="vads-u-margin-y--1">
-          <Link to="/" onClick={handleClick()}>
-            Review your appointments
-          </Link>
+          <va-link
+            href="/"
+            onClick={handleClick()}
+            text="Review your appointments"
+            data-testid="review-appointments-link"
+          />
         </div>
         <div>
-          <Link to="/new-appointment">Schedule a new appointment</Link>
+          <va-link
+            href="/new-appointment"
```

Comment:
This link is going to the wrong path. "/new-appointment" was enrolled in a special routes file. I don't know how it works but @vbahinwillit may have the background on it.